spec/default_facts.yml

```diff
@@ -1,12 +1,6 @@
-# This file is managed via modulesync
-# https://github.com/voxpupuli/modulesync
-# https://github.com/voxpupuli/modulesync_config
+# Use default_module_facts.yml for module specific facts.
```

Comment:
Please undo the changes here, otherwise our https://github.com/voxpupuli/modulesync_config will overwrite it on the next run and creates an unneeded diff.